Edits (#160)
mulesoft/docs-gateway: policies/modules/ROOT/pages/policies-included-apply.adoc

:keywords: policy, endpoint
:page-aliases: api-manager::using-policies.adoc

> In Flex Gateway Connected Mode and Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.

Reviewer: ```suggestion
For Flex Gateway running in Connected Mode, or for Mule Gateway, you apply included policies via *API Manager*. The following steps demonstrate this.
```

> In Flex Gateway Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Flex Gateway Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].

Reviewer: ```suggestion
For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Probably don't need the second "Flex Gateway" since we've already said "Flex Gateway running in Local Mode" in the previous sentence. It seemed redundant.

> For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Flex Gateway Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].

Reviewer: ```suggestion
For Flex Gateway running in Local Mode, you apply policies via an `ApiInstance` YAML resource. For information about applying policies in Local Mode, see the xref:gateway::flex-local-configuration-reference-guide.adoc#api-instance[Declarative Configuration Reference Guide].
```

Either rename that second mention of "Flex Gateway" so that it matches the naming of the instance in the previous sentence, or omit it.

We need to be consistent with how we refer to Flex Gateway running in the two modes.

To apply a policy for Mule Gateway, a Mule application must first be deployed with an xref:connectors::http/http-listener-ref.adoc[HTTP-based or HTTPS-based flow] that is linked to the API through autodiscovery.
====

> To apply a policy for Flex Gateway Connected Mode or Mule Gateway:

Reviewer: ```suggestion
To apply a policy for Flex Gateway running in Connected Mode, or for Mule Gateway:
```

> To apply a policy for Flex Gateway running in Connected Mode or Mule Gateway:

Reviewer: ```suggestion
To apply a policy for Flex Gateway running in Connected Mode, or for Mule Gateway:
```

Otherwise it implies that Flex Gateway is running in Mule Gateway.

. Go to *Anypoint Platform > API Manager*.
. In *API Administration* click the name of the API to which to apply a policy.
. From the left navigation menu, click *Policies*.

> . Click *+ Add policy*

Reviewer 2: ```suggestion
. Click *+ Add policy*.
```

> . Select the policy to apply

Reviewer 2: ```suggestion
. Select the policy to apply.
```
